### PR TITLE
DE39021 - Adding a display capture which is required for Bongo

### PR DESCRIPTION
**Pull request by pbrown-d2l**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ Parameters:
  * `allowFullScreen`: whether the frame can be placed into full screen mode, `false` by default
  * `allowMicrophone`: whether the frrame will allow access to the microphone, `false` by default
  * `allowCamera`: whether the frrame will allow access to the camera, `false` by default
+ * `allowScreenCapture`: whether the frrame will allow access to the camera, `false` by default
```
Comment (dbatiste) [inline README.md:70]: Looks like frrame has been spelt wrong a few times.  Lol.  :)

Comment (pbrown-d2l) [reply README.md:70]: damn copy pasta. I will fix that

**Comment by pbrown-d2l** [comment]
This should be good to go. I unfortunately can't merge or release a new version to be used in the LMS.

**Comment by dlockhart** [comment]
Done -- give it 5 mins and it should be published to NPM.

**Pull request merged**